Formula/f/falcosecurity-libs.rb

```diff
@@ -0,0 +1,64 @@
+class FalcosecurityLibs < Formula
+  desc "Foundational libraries that constitute the core of Falco's functionality"
```

Comment:
Not sure what is good description for this. Currently grabbed snippet from https://github.com/falcosecurity/evolution#core
> Foundational libraries that constitute the core of Falco's functionality, offering essential features including kernel drivers and eBPF probes.

Alternative could be Debian's description of `Core libraries for Falco and Sysdig`